modules/govuk_jenkins/templates/jobs/copy_data_to_integration.yaml.erb

```diff
@@ -60,6 +70,11 @@
             HOSTNAME=deploy.integration.publishing.service.gov.uk
             API_KEY=<%= @ci_alphagov_api_key %>
             AUTH_TOKEN=<%= @auth_token %>
+      - slack:
+          team-domain: <%= @slack_team_domain %>
+          auth-token: <%= @environment_variables['SECOND_LINE_SLACK_AUTH_TOKEN']%>
```

Comment:
Yup, that's exactly it. It worked well in integration, and then we removed the environment variable from integration and staging so it doesn't get too spammy.